### ISSUE TITLE
Orphan branches silently ignored

### ISSUE THREAD

**Issue by jan-dolejsi** [0xABAD/behavior_tree#4]
This allows orphan branches to be silently ignored:

https://github.com/0xABAD/behavior_tree/blob/47005b56a8796613df3290d11cc260a85aa3ce6d/btree.js#L214

**Comment by 0xABAD** [comment]
Can you provide an example?  I'm not following correctly from what you are referencing here.

**Comment by jan-dolejsi** [comment]
Sure, one of the examples:

```ascii
->
| | (condition)
```

... which is an obvious error in this case, but when nested somewhere deeper in the tree can be harder to find.

This test is now checking for such model error:

```javascript
        it('returns error for double indent (orphan tree branches)', () => {
            let tree = btree.parse(`->
                |   |   (some condition)
                `);
            expect(tree.root).to.equal(null);
            expect(tree.error).to.contain("indentation", "there should be error message");
        });
```

This is fixed in the #6 .

**Comment by 0xABAD** [comment]
Thanks.
